src/components/shared/Title.jsx

```diff
@@ -0,0 +1,11 @@
+import Typography from "@mui/material/Typography";
+
+const Title = ({ children }) => {
+  return (
+    <>
+      <Typography variant="h5">{children}</Typography>
+    </>
```

Comment:
```suggestion
      <Typography variant="h5">{children}</Typography>
```
태그 하나만 둘거면 프레그먼트가 필요없음